tests/data_files/Makefile

```diff
@@ -1854,14 +1942,22 @@ list_intermediate:
 # third-party tools.
 list_final:
 	@printf '%s\n' $(all_final) | sort
-.PHONY: list_intermediate list_final
+list_parse_input:
+	@printf '%s\n' $(all_parse_input) | sort
+.PHONY: list_intermediate list_final list_parse_input
 
 ## Remove intermediate files
 clean:
 	rm -f $(all_intermediate)
 ## Remove all build products, even the ones that are committed
 neat: clean
-	rm -f $(all_final)
-.PHONY: clean neat
+	rm -f $(shell ls $(all_final) | grep -v -f uncategorized.lst)
```

Comment:
> Remove all build products, even the ones that are committed

Can we change this comment now to specify what is happening with uncategorized.lst?

Reply:
Will update it after preceding PR merged.